src/main/java/org/frc5687/infiniterecharge/robot/Constants.java

```diff
@@ -7,7 +7,7 @@
 public class Constants {
     public static final int  TICKS_PER_UPDATE = 10;
     public static final double METRIC_FLUSH_PERIOD = 1.0;
-    public static final double UPDATE_PERIOD = 0.01;
+    public static final double UPDATE_PERIOD = 0.02;
```

Comment:
Changing the update_period will impact your trajectories!  Be sure you've regenerated them!